openerp/addons/base/ir/ir_cron.py

```diff
@@ -87,7 +87,7 @@ def method_direct_trigger(self, cr, uid, ids, context=None):
             context={}
         cron_obj = self.browse(cr, uid, ids, context=context)
         for cron in cron_obj:
-            self._callback(cr, uid, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
+            self._callback(cr, SUPERUSER_ID, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
```

Comment:
Yep, we often need to duplicate a cron and assign them to different users assigned to different companies so record rules and properties are respected.